Source/NSPasteboard.m

```diff
@@ -553,6 +553,15 @@ to a filter implemented using the standard method (of
 #import "GNUstepGUI/GSServicesManager.h"
 #import "GNUstepGUI/GSPasteboardServer.h"
 
+/*
+ * FIXME
+ * We should learn to handle 
+ * NSPasteboardTypePNG
```

Comment:
I do not know where the right place for this FIXME is because I still do not understand in depth how NSPasteboard functions and where this would need to be implemented. I placed it there because I thought it would be visible since NSPasteboard.m is an entrypoint for anyone trying to implement something.